doc/ref_arch/openstack/chapters/chapter03.md

```diff
@@ -253,31 +250,19 @@ The physical resources required for the NFVI are mainly based on COTS X86 hardwa
 HW profiles are defined in the chapters 5.3 and 5.4 of the reference model document.
 
 #### 3.4.2.2. Network
-(Spine-Leaf, East/West and North-South Traffic)
-
-NFVI Network & Security
-
-the recommended network architecture is spine and leaf topology however for small sites a legacy topology (access/aggregation switches) can be setup.
-
-Content to be developed along the following lines
-- do we include FW to separate control vs data plane?
-- do we include DC GW to separate NFVI from external environments?
-- do we include OoB switch
-
-Each instance of IaaS relies on physical resource: servers, switches, ToR
+The recommended network architecture is spine and leaf topology however for small sites a legacy topology (access/aggregation switches) can be setup. Each instance of IaaS relies on physical resource: servers, switches, ToR
```

Comment:
I'm not sure what this bit "Each instance of IaaS relies on physical resource: servers, switches, ToR" is intending to add here.  Is the intent to express that the IaaS network fabric design has to encompass the NICs in the servers, the leaf/spine switches, etc?

Reply:
@kedmison If the intent is not clear please suggest changes. Thanks
@samuelhellec Thoughts?

Reply:
How about:
"The recommended network architecture is spine and leaf topology; however, for small sites, a legacy topology (access/aggregation switches) can be set up. "

I propose dropping the sentence "Each instance of IaaS relies on physical resource: servers, switches, ToR".  It may be just me but I don't perceive it as adding value and so in the spirit of simplification I would remove it.

Reply:
@kedmison Thanks.
@samuelhellec Are you OK with the proposed change or would like to add something?

Reply:
agreed

Reply:
@kedmison @samuelhellec Change made. Please review. Thanks